src/lorawan_app.erl

```diff
@@ -84,9 +84,20 @@ start(_Type, _Args) ->
                 #{env => #{dispatch => Dispatch},
                 stream_handlers => [lorawan_admin_logger, cowboy_compress_h, cowboy_stream_h]})
     end,
-    lorawan_sup:start_link().
+    ExtList = case application:get_env(extensions) of
+        undefined ->
+            undefined;
+        {ok, undefined} ->
+            undefined;
+        {ok, List} ->
+            List
+    end,
+    ok = start_extensions(ExtList),
+    {ok, SupPid} = lorawan_sup:start_link(),
```

Comment:
On some other thought about race conditions and etc. This sup startup should possibly be moved a notch higher. To appear right before the extensions case. (Where by github the old sup start was).